README.md

```diff
@@ -37,4 +37,26 @@ Praharsh Patel </br>
     <li> Straight line drawing </li>
 </ul>
 
+## Version 0.2.0
+### Features
+<ul>
+    <li> Added a button to allow drawing a straight line</li>
+    <li> Added a button to allow the user to select an image</li>
+    <li> Added ability to detect whether the file is an image type or not</li>
+</ul>
+
+### Bug Fixes
+<ul>
+    <li> <em>Image and tool bar are seprate, but previosly they were overlapping.</em> </li>
```

Comment:
Can you not make these 3 points italicized

Reply:
Yes, I have made the commit to change it to be normal text and not italicized.